redfish-core/include/event_service_manager.hpp

```diff
@@ -1078,6 +1091,23 @@ class EventServiceManager
             }
         }
     }
+    void sendBroadcastMsg(const std::string& broadcastMsg)
```

Comment:
does broadcast message is disabled in 1050 code?

Reply:
no it was there, but seems handled differently